eva_sub_cli_processing/process_jobs.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime
+
+import requests
+from ebi_eva_common_pyutils.common_utils import pretty_print
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import AppLogger
+from retry import retry
+
+
+def _url_build(*args, **kwargs):
+    url = cfg.query('submissions', 'webservice', 'url') + '/' + '/'.join(args)
+    if kwargs:
+        return url + '?' + '&'.join(f'{k}={v}' for k, v in kwargs.items())
+    else:
+        return url
+
+
+@retry(tries=5, backoff=2, jitter=.5)
+def _get_submission_api(url):
+    auth = (cfg.query('submissions', 'webservice', 'admin_username'), cfg.query('submissions', 'webservice', 'admin_password'))
+    response = requests.get(url, auth=auth)
+    response.raise_for_status()
+    return response.json()
+
+
+@retry(tries=5, backoff=2, jitter=.5)
+def _put_submission_api(url):
+    auth = (cfg.query('submissions', 'webservice', 'admin_username'), cfg.query('submissions', 'webservice', 'admin_password'))
+    response = requests.put(url, auth=auth)
+    response.raise_for_status()
+    return response.json()
```

Comment:
This is fine for the initial version but I would reiterate what I wrote in the second half of [this comment](https://github.com/EBIvariation/eva-sub-cli/pull/40#discussion_r1636377305) about a client class for the submission-ws.  This way all our Python code can use the web service uniformly, just configured using admin credentials when necessary.